src/unit_tests/analysisd/test_decoder_syscollector.c

```diff
@@ -1377,7 +1413,7 @@ void test_syscollector_dbsync_network_protocol_valid_msg(void **state)
 
     assert_int_not_equal(ret, 0);
 }
-void test_syscollector_dbsync_network_address_valid_msg(void **state)
+void test_syscollector_dbsync_network_address_invalid_msg(void **state)
```

Comment:
Since we named a test as "test_syscollector_dbsync_network_address_valid_msg_inserted" I would add the "modified" word to this test name.